packages/snap-preact-components/src/components/Trackers/SearchResultTracker/SearchResultTracker.tsx

```diff
@@ -49,6 +49,8 @@ export const SearchResultTracker = observer((properties: SearchResultTrackerProp
 		>
 			{children}
 		</div>
+	) : (
+		<Fragment></Fragment>
```

Comment:
I wonder if it makes sense to return the children in the fragment? Consumers may not know it is not setup properly if we did I suppose eh?

Reply:
yeah I think your right it probably should return the children. If want to inform of something wrong a console warning is better than breaking and not returning anything